CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Fix tests for interactive prompting.
```

Comment:
Agreed, since it is not a user facing change, maybe it shouldn't be in the changelog. Also, I think those tests that are fixed might have been created somewhere earlier in this branch.

Reply:
I removed this text.